From 4991ea943a4a5051c35c75bea8fd013bf0658c6b Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Tue, 8 Sep 2015 15:25:42 +0800
Subject: [PATCH 0409/5242] MLK-11520 ARM: imx: enable axi clk in clk_init_on
 on i.mx6qdl

commit  608b8994d984892c14aa30c6d6dfb461c30b3a60 from
https://source.codeaurora.org/external/imx/linux-imx.git

AXI clk should be always enabled, as many peripheral
devices need this clk to be on, especially for busfreq,
AXI podf change needs AXI clk and its parent on.

Signed-off-by: Anson Huang <b20788@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx6q.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/imx/clk-imx6q.c b/drivers/clk/imx/clk-imx6q.c
index ea2b208..0f1414c 100644
--- a/drivers/clk/imx/clk-imx6q.c
+++ b/drivers/clk/imx/clk-imx6q.c
@@ -105,6 +105,7 @@
 	IMX6QDL_CLK_ROM,
 	IMX6QDL_CLK_ARM,
 	IMX6QDL_CLK_OCRAM,
+	IMX6QDL_CLK_AXI,
 };
 
 static struct clk_div_table clk_enet_ref_table[] = {
-- 
1.7.9.5

